src/wallet/test/wallet_tests.cpp

```diff
@@ -413,6 +414,14 @@ BOOST_FIXTURE_TEST_CASE(rescan, TestChain100Setup)
         UniValue response = importmulti(request);
         BOOST_CHECK_EQUAL(response.write(), strprintf("[{\"success\":false,\"error\":{\"code\":-1,\"message\":\"Failed to rescan before time %d, transactions may be missing.\"}}]", newTip->GetBlockTimeMax()));
     }
+
+    // Verify ScanForWalletTransactions does not return null when the scan is
+    // elided due to the nTimeFirstKey optimization.
+    {
+        CWallet wallet;
```

Comment:
Thanks, guess travis is still useful for some things. Fixed in d7e96f8562680ec30e1ca38f72206179693ab753